docs/question/accessibility.md

```diff
@@ -0,0 +1,80 @@
+# Authoring accessible questions
+
+Ensuring that PrairieLearn questions are accessible to all students, including those with disabilities, is crucial for an inclusive learning environment. This guide provides recommendations for authors to create accessible question content.
+
+## Semantic HTML
+
+Using semantic HTML elements appropriately is the foundation of web accessibility. Semantic HTML means using HTML tags for their intended purpose, which helps assistive technologies (like screen readers) understand the structure and meaning of the content.
+
+- **Headings**: Use `<h1>`, `<h2>`, etc., to structure your question content logically. Avoid skipping heading levels.
```

Comment:
Do we really want `<h1>`? Since the question panel header already uses `<h1>`, maybe suggest it to start at `<h2>`?

Reply:
Interesting point! I think I'd rather err on the side of a simple suggestion that doesn't require complex background knowledge about the overall structure of the page. While a single `<h1>` is of course ideal if one controls all the content on a page, I don't think it's actually completely invalid to have multiple `<h1>` tags, and at least for accessibility, screen readers will handle that fine.

Does that sound ok?